index.html

```diff
@@ -1,13 +1,41 @@
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
   <title>DeVinci App</title>
   <meta name='author' content='Patrick Friedrich'>
   <meta name='description' content='Users can have their truly personal AI assistant on this web app. The AI model runs on the user`s device (via the browser) and does not rely on any centrally hosted services.'>
+  <meta name="keywords" content="AI Chat, Conversational AI, Devinci AI, Chatbot, AI Assistant">
```

Comment:
we should decide on one consistent way of writing DeVinci;
we've got DeVinci, deVinci, Devinci across the app. Another option could be dVinci.
Which one do you think is best?

Reply:
sounds good, cheers, let's stick with DeVinci then for now. There's already a bit SEO around it